internal/cmd/cmd_set.go

```diff
@@ -203,14 +209,14 @@ func evalSET(c *Cmd, s *dstore.Store) (*CmdRes, error) {
 	// XX: only set the key if it already exists
 	// So, if it does not exist, we return nil and move on
 	if params[types.XX] != "" && existingObj == nil {
```

Comment:
The current implementation incorrectly returns "1" (success) for cases where the SET operation was NOT performed. This violates the intended semantics where:

1 should mean "1 key was successfully set/updated"
0 should mean "0 keys were set/updated" (not an error)